resources/user-agents/apps/microsoft/bing-search-app-android-2.json

```diff
@@ -0,0 +1,51 @@
+{
+    "division": "Bing Search App #MAJORVER#.#MINORVER# for Android",
+    "versions": ["2.1", 2],
+    "sortIndex": 485,
+    "lite": false,
+    "standard": true,
+    "userAgents": [
+        {
+            "userAgent": "Bing Search App #MAJORVER#.#MINORVER# for Android",
+            "browser": "bing search app",
+            "engine": "Blink",
```

Comment:
These are all "WebKit" in  this file, should probably change to "WebKit" here to reduce lines in output file some (until  #1723 is implemented).

Reply:
fixed